README.md

```diff
@@ -4,13 +4,37 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 # beman.exemplar: A Beman Library Exemplar
 
-![Continuous Integration Tests](https://github.com/bemanproject/exemplar/actions/workflows/ci_tests.yml/badge.svg)
+<div style="display: flex; align-items: center;">
+  <img
+    src="https://github.com/bemanproject/beman/blob/main/images/logos/beman_logo-beman_library_under_development.png"
+    style="width: 10%; height: auto;"
+    alt="Beman Logo"
+  >
+  <table>
+    <tr>
+      <td>
+        <img
+          src="https://github.com/bemanproject/exemplar/actions/workflows/ci_tests.yml/badge.svg"
+          alt="Continuous Integration Tests"
+        >
+      </td>
+      <td>
+        <img
+          src="https://github.com/bemanproject/exemplar/actions/workflows/pre-commit.yml/badge.svg"
+          alt="Lint Check (pre-commit)"
+        >
+      </td>
+    </tr>
+  </table>
+</div>
```

Comment:
To have the PR merged, can we omit the dragon icon for now?

Also, I don't see the point of using a table here. Having two badges one by one are perfectly fine.

HTML may not be universally supported by all markdown render, it's hard for linter to check over. They add clutter for editing.

```suggestion
![Continuous Integration Tests](https://github.com/bemanproject/exemplar/actions/workflows/ci_tests.yml/badge.svg)
![Lint Check (pre-commit)](https://github.com/bemanproject/exemplar/actions/workflows/pre-commit.yml/badge.svg)
```